packages/web/pages/roles.tsx

```diff
@@ -42,115 +58,116 @@ type CardProps = {
-  comingSoon?: boolean;
+  difficulty: string;
+  time: string;
 };
 
 const Card: React.FC<CardProps> = ({
   title,
   description,
   link,
   icon,
-  progress,
-  completed,
-  color,
-  comingSoon,
+  difficulty,
+  time,
 }) => {
-  const spin = keyframes`
-    from { width: 0; }
-    to { width: ${progress}%; }
-  `;
+  let difficultyBgColor;
+  switch (difficulty) {
+    case Difficulty.HARD:
+      difficultyBgColor = '#e53e3e87';
+      break;
+    case Difficulty.MEDIUM:
+      difficultyBgColor = '#d69e2e8a';
+      break;
+    default:
+      difficultyBgColor = '#38a16987';
+  }
+
+  let timeBgColor;
+  switch (time) {
+    case Time.LONG:
+      timeBgColor = '#e53e3e87';
+      break;
+    case Time.MEDIUM:
+      timeBgColor = '#d69e2e8a';
+      break;
+    default:
+      timeBgColor = '#38a16987';
+  }
 
   return (
-    <MetaLink
-      display="flex"
-      borderRadius="lg"
-      borderWidth="1px"
-      textColor="white"
-      alignItems="center"
-      textAlign="center"
-      placeContent="center"
-      maxW="sm"
-      minH="3xs"
-      w={['full', 'auto']}
-      pointerEvents={comingSoon ? 'none' : 'auto'}
-      cursor={comingSoon ? 'default' : 'pointer'}
-      href={comingSoon ? '' : link}
-      sx={{
-        bgColor: '#110035',
-        borderColor: 'whiteAlpha.400',
-        transition: 'all 0.1s ease-in-out',
-        _hover: { bgColor: '#150042', borderColor: 'whiteAlpha.700' },
-      }}
-    >
-      {comingSoon && (
-        <Box position="absolute" zIndex={2}>
-          <Heading fontSize="lg">COMING SOON</Heading>
-        </Box>
-      )}
-      <Box
-        borderRadius="lg"
-        position="relative"
-        p={{ base: 4, xl: 8 }}
-        sx={{
-          '&::after': comingSoon
-            ? {
-                content: '" "',
-                position: 'absolute',
-                zIndex: 1,
-                borderRadius: 6,
-                display: 'flex',
-                height: 'full',
-                width: 'full',
-                top: 0,
-                left: 0,
-                right: 0,
-                background: 'rgba(63, 68, 94, 0.7)',
-              }
-            : {},
-        }}
-      >
-        {typeof completed === 'number' && (
-          <Box>
-            <Text fontFamily="heading" textColor={color}>
-              Completed: {completed}
-            </Text>
-          </Box>
-        )}
-        {progress && (
-          <Box
-            background="rgba(255,255,255,0.1)"
-            justifyContent="flex-start"
-            borderRadius="100px"
+    <Link role="group" _hover={{ textDecoration: 'none' }} href={link}>
+      <MetaTile height="full" width="full">
+        <MetaTileHeader>
+          <Flex
+            flexDir="column"
+            gap={1}
+            pos="absolute"
+            left={-8}
+            py={3}
+            px={4}
+            top={-8}
+            background="rgba(255, 255, 255, 0.1)"
+            backdropFilter="blur(10.5px)"
```

Comment:
I didn't think there was a defined prop for this:

```suggestion
            sx={{ backdropFilter: 'blur(10.5px)' }}
```